### ISSUE TITLE
Hide Completed/Show All has a bug and its fix

### ISSUE THREAD

**Issue by bhushitjoshipura** [meteor/blaze-tutorial#28]
Hide Completed/Show All as shown in tutorial has a bug.

1. Check a task
2. Press "Hide Completed"
3. The task doesn't show
4. Press "Show all"
5. The task shows up
6. Bad, current behavior: The task checkbox is displayed unchecked, leading the user to think the task is not checked - and mismatches unchecked tasks with the pending count (by one, in this case)
7. Correct, future behavior: The task checkbox is checked

The fix lies in imports/ui/Task.html
Instead of:
`<input type="checkbox" class="toggle-checked" />`
or similar code, 
Mention:
`        {{#if isChecked}}
            <input type="checkbox" checked class="toggle-checked"/>
        {{else}}
            <input type="checkbox" class="toggle-checked" />
        {{/if}}`

**Comment by bhushitjoshipura** [comment]
Is anyone paying attention to this ticket? Is there some way for me to directly offer the code? (This is a tutorial repository - so the patch also has to be explained - that is what holds me back...)

**Comment by filipenevola** [comment]
Hi @bhushitjoshipura, yes, we are paying attention, we already replied many issues opened by you. Thank you 😉 

And yes, if you found a bug you can contribute with the solution as explained here https://github.com/meteor/blaze-tutorial#making-a-pull-request

TL;DR: the source code is also in this repository.
